app/javascript/shared/components/tags.jsx

```diff
@@ -36,14 +36,6 @@ class Tags extends Component {
       prevLen: 0,
       showingRulesForTag: null,
     };
-
-    const algoliaId = document.querySelector("meta[name='algolia-public-id']")
```

Comment:
I'm assuming we're not removing these `<meta /> tags from markup in `_top.html.erb` yet because other things still rely on Algolia?

```erb
      <meta name="algolia-public-id" content="<%= ApplicationConfig["ALGOLIASEARCH_APPLICATION_ID"] %>">
      <meta name="algolia-public-key" content="<%= ALGOLIASEARCH_PUBLIC_SEARCH_ONLY_KEY %>">
```

Reply:
Yeah